tests/payment-splitter.tests.ts

```diff
@@ -0,0 +1,56 @@
+import {bnArg, expect, fromSigner, setupContract} from './helpers'
+import {network, patract} from 'redspot'
+const { api } = network;
+
+const {getRandomSigner} = patract
+
+describe('MY_PAYMENT_SPLITTER', () => {
+    async function setup() {
+        const ian = await getRandomSigner()
+        const kayne = await getRandomSigner()
+        let contract = await setupContract('my_payment_splitter', 'new', [kayne.address, ian.address], [40, 60]);
+
+        return { contract, kayne, ian }
+    }
+
+    it('PAYMENT SPLITTER - init values', async () => {
+        // Arrange - Create a contract
+        const {
+            contract,
+            kayne,
+            ian,
+        } = await setup()
+
+        // Assert - Init with shares
+        await expect(contract.query.shares(kayne.address)).to.have.output(40)
+        await expect(contract.query.shares(ian.address)).to.have.output(60)
+        await expect(contract.query.totalShares()).to.have.output(100)
+        await expect(contract.query.payee(0)).to.have.output(kayne.address)
+        await expect(contract.query.payee(1)).to.have.output(ian.address)
+    })
+
+    it('PAYMENT SPLITTER - release a native token', async () => {
+        // Arrange - Create a contract
+        const {
+            contract,
+            kayne,
+            ian,
+        } = await setup()
+
+        // Act - Send native token and release them
+        await expect(contract.contract.query.totalReleased()).to.have.output(0);
+        await expect(fromSigner(contract.contract, contract.alice.address).tx.receive({ value: 20000000 })).to.eventually.be.fulfilled
+        await expect(contract.contract.tx.release(kayne.address)).to.eventually.be.fulfilled
```

Comment:
This is a general comment to tests.
I don't think we need any expect in Act section. Moreover await expect(...).to.eventually.be.fulfilled is redundant, as test will fail if promise will be rejected anyway.

Reply:
Yes, but it is more readable for people whose is not familiar with js.